voxblox_ros/src/tsdf_server.cc

```diff
@@ -80,7 +80,7 @@ TsdfServer::TsdfServer(const ros::NodeHandle& nh,
   tsdf_slice_pub_ = nh_private_.advertise<pcl::PointCloud<pcl::PointXYZI> >(
       "tsdf_slice", 1, true);
 
-  nh_private_.param("pointcloud_queue_size_", pointcloud_queue_size_,
```

Comment:
sry about that, that was probably me when I refactored the ros params...

Reply:
90% chance it was there before the refactor ;)